exercises/Scientific-tree-biomass-Python.md

```diff
@@ -21,15 +21,15 @@ in a 96 hectare area of the Western Ghats in India.
 
 1.  Write a function that takes an array/Series of tree diameters as an
     argument and returns an array/Series of tree masses.
-2.  [The raw data](http://esapubs.org/archive/ecol/E091/216/Macroplot_data_Rev.txt)
+2.  [The raw data](http://esapubs.org/archive/ecol/E091/216/Macroplot-data-Rev.txt)
```

Comment:
Changes like this aren't correct because they are links to data that we don't control the file names of.